test/tester.go

```diff
@@ -161,6 +175,8 @@ func (at *AccountsTester) Post(endpoint string, params url.Values, postParams ur
 }
 
 // Put executes a PUT request against the test service.
+//
+// NOTE: The Body of the returned response is already read and closed.
```

Comment:
I don't know what I'll need, so I'd rather have the entire response. Usually it's the status and the headers.

Reply:
I'm not sure what I think about this one.

On one hand, yes, we don't need the entire response because we already have the body and there isn't anything else that's useful in the response besides status code and headers. 

On the other hand, passing a single pointer to the response is a non-issue, as long as the developer realises that they already have the body conveniently returned to them and doesn't try to read it again. Changing the return signature to `(code int, head http.Header, body []byte, err error)` doesn't seem to be improving it much.

To me it looks like busywork (because we'll need to fix literally all tests) with no real impact. But if you find this valuable I'll make the change.

Reply:
https://github.com/SkynetLabs/skynet-accounts/issues/111